### DISCUSSION TITLE
collect/

### DISCUSSION THREAD

**Discussion by giscus** [MarsperL/MarsperL.github.io#18] [Announcements]
# collect/

#article-container a:not(.headerlink, .fancybox, .default-style a) {     font-weight: 700;     color: var(--font-color);     padding: 0 3px;     border-bottom: 2px var(--leonus-main) solid; }  #artic

http://localhost:4000/collect/

**Comment by MarsperL** [comment]
12332

**Comment by MarsperL** [comment]
1233313

**Comment by MarsperL** [comment]
213233131332

**Comment by MarsperL** [comment]
123232131321323

**Comment by MarsperL** [comment]
的覆盖广泛的
